### PR TITLE
Lesson1 3

### PR DESCRIPTION
**Pull request by omer-jan**
I have completed this week assignment. please review my code and let me know your comments and suggestions.

### CODE REVIEW

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react"; 
+let AddTodoForm=(props)=>{
+let handleAddTodo=(e)=>{
+    e.preventDefault();
+    let todoTitle=e.target.title.value;
+    console.log(todoTitle);
+    props.onAddTodo(todoTitle);
+    e.target.title.value="";
```
Comment (kwilson05) [inline src/AddTodoForm.js:8]: Only think I'll suggest is try using `form.reset()`. I think you'll figure out how to use this

**Review by kwilson05 — APPROVED** [review]
Looks like you know what you're doing Omer lol 🎖️ 💯 Nice job man!!

I want you to take a look at the change I suggested but go ahead and merge after that. 
Keep up the good work, Omer

**Comment by kwilson05** [comment]
Sorry, I called you Omer but your name is `Mohammad`. Thanks is my fault!! 🙏

File: src/AddTodoForm.js (hunk 1 above)
Comment (omer-jan) [reply src/AddTodoForm.js:8]: thank you for your suggestion. 
e.target.reset();

**Pull request merged**